docs/app-dev-context/cluster-install.md

```diff
@@ -0,0 +1,73 @@
+---
+id: cluster-install
+title: How to quickly install a Temporal Cluster for testing and local development
+sidebar_label: Quick install
+description: There are multiple ways to quickly install and run a Temporal Cluster.
+---
+
+Whenever you are developing Temporal Applications, you'll want to have a [Temporal Cluster](/concepts/what-is-a-temporal-cluster) up and running.
+You can interact with a Cluster through [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl) commands.
```

Comment:
Let's rename this to "local development server" instead of "cluster", for something to be a cluster it needs to be a "group of things".
Using the term cluster in the first step to getting started is a major turn off for new users IMO.

Reply:
Also, as Temporalite matures we'll probably change Gitpod to use it too so the only "cluster" we'll have is using Docker compose.
One last thing, docker-compose will soon be able to run using sqlite so it might also turn into an all-in-one process solution.

Reply:
>Let's rename this to "local development server" instead of "cluster", for something to be a cluster it needs to be a "group of things".
Using the term cluster in the first step to getting started is a major turn off for new users IMO.

Respectfully - no, we will continue using the term Cluster. Your opinion is valid - but consistency matters way more here.